### PR TITLE
Improve Feedback #22

### LINKED ISSUE #22: Improve Feedback
Show Spinner when solving exercise and resetting
Hide terminal on file-check
(Continuous Feedback on terminal command execution)

### PR DESCRIPTION
**Pull request by LukasBoll**
removed Console output from CheckFileList
locks screen while solving/resetting is in progress
closes #22

### CODE REVIEW

File: react-app/src/components/StepperComponent.tsx [hunk 1]
```diff
@@ -193,6 +202,40 @@ const StepperComponent = (props: StepperComponentProps) => {
...
+            style: {
+              backgroundColor: 'transparent',
+              boxShadow: 'none',
+              overflow: 'visible',
+            },
+          }}
+          open={true}
+          maxWidth={false}
+        >
+          <ClipLoader
```
Comment (eneufeld) [inline react-app/src/components/StepperComponent.tsx]: why not use https://material-ui.com/components/progress/ ?

**Comment by eneufeld** [comment]
Thank you. Looks good overall. Will approve and merge after clarification.

File: react-app/src/components/StepperComponent.tsx (hunk 1 above)
Comment (LukasBoll) [reply react-app/src/components/StepperComponent.tsx]: I just wasn’t aware of the component from Material-Ui. I will replace the current Loader. Thank you.

**Review by eneufeld — APPROVED** [review]
Thank you!

**Pull request merged**